README.md

```diff
@@ -132,16 +132,17 @@ $ yarn install
 $ rake dev:setup
 $ heroku local -f Procfile.dev
 # if you run into the following error:
-```
```

Comment:
Maybe the better fix here is
```
$ heroku local -f Procfile.dev
```
if you run into the error: `bash: heroku: command not found` visit ...